JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/CompileAndRunBean.java

```diff
@@ -123,9 +125,12 @@ public String run(String projectHash) {
                     mes += "Stderr: " + help + "\n";
```

Comment:
I've only noticed that you have initiated BufferedReader and BufferedWriter, but then you read directly from InputStream (???). I did not get this trick.